ZupIT/ritchie-cli
# Feature/add formula rit list formulas

### Description

Added command `rit list formula` (#857)
* Command available for prompt and input flags execution.
* Unit tests created for both cases.
* Formulas are listed based on selected repository or all repositories.
### How to verify it
* Run `rit list formula` and follow prompt instructions.
* Run `rit list formula --name=<repo_name>` to check for input flag compatibility.
* Run `rit lit formula --name=ALL` to list formulas from all repositories when using input flags.

Prompt execution
![image](https://user-images.githubusercontent.com/60020008/115717506-9eb05700-a350-11eb-96df-236a3af48222.png)

Results for selected repository
![image](https://user-images.githubusercontent.com/60020008/115716827-fa2e1500-a34f-11eb-81fc-4c2ab2387272.png)
### Changelog

Added command `rit list formula`
